xml/art-obs-beginners-guide.xml

```diff
@@ -1006,30 +1056,56 @@ Patch0:         &gitproject;_main.diff</screen>
     <para>Create the branch from &exampleuser_plain;'s home project:</para>
     <screen>&prompt.user2;<command>osc</command> bco &obshome1; &gitproject;</screen>
     <para>This will create a branched package in &obsa; at <uri>&obsbranch2;/&gitproject;</uri>
-     and will checkout a directory <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.</para>
+     and will check out a directory
+     <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.
```

Comment:
I know. 😄 

The slashes (instead of the colons) are used on purpose and are explained in "Section 4 Configuring Your System for OBS" (basically setting `checkout_no_colon=1`). IMHO it think this layout is be a bit easier to follow for users and they don't mix it up (checking out a directory that contains colons?).